src/main/java/de/eldoria/pickmeup/PickMeUp.java

```diff
@@ -57,4 +65,11 @@ public void onReload() {
         onDisable();
         onEnable();
     }
+    public static PickMeUp instance(){
+        return instance;
+    }
+    public static NamespacedKey offsetterIdentifierKey(){
+        return offsetterIdentifierKey;
+    }
```

Comment:
The namespaced key should be part of the class where it is used as a constant. Especially since it is exclusively used by only one class.
you can also safely use the deprecated constructor since spigot is just scared people would use it in the wrong way.

`new NamespacedKey("pickmeup", "IsOffsetter")`